src/renderer/api-decorator.js

```diff
@@ -444,8 +453,14 @@ limitations under the License.
             }, 1);
         });
 
-        var convertedOpts = convertOptionsToElectronSync(options);
-        fin.__internal_.downloadPreloadScripts(convertedOpts.preload, () => {
+        const convertedOpts = convertOptionsToElectronSync(options);
+        const preloadScriptsPayload = {
+            uuid: options.uuid,
+            name: options.name,
```

Comment:
Wouldn't it be safer to use `winOpts.uuid` and `winOpts.name`... ? I'm not sure how this `createChildWindow` is being called or from where, but are we sure both `options.uuid` and `options.name` are available?